tests/serenity/src/test/java/bla/tm/definitions/site/products_and_docs/PD_APIExplorerDefinition.java

```diff
@@ -10,6 +11,7 @@
 
 public class PD_APIExplorerDefinition {
 
+    private UserData admin = new UserData("UserData", "1234567");
```

Comment:
its not a good practice to store data in java file
could you create separate data file for it?